### PR TITLE
Minor edits for clarity

### PR DESCRIPTION
**Pull request by dmatrix**
## Why are these changes needed?

Minor edits were discovered during proofreading and testing the end-to-end example

## Checks

- [ x] I've run `scripts/format.sh` to lint the changes in this PR.

### CODE REVIEW

File: doc/source/serve/deployment-graph.md [hunk 1]
```diff
@@ -39,37 +44,38 @@ __[Full Ray Enhancement Proposal, REP-001: Serve Pipeline](https://github.com/ra
...
     - A deployment node is created from `@serve.deployment` decorated function or class. 
     - You can construct different deployment nodes from same class or function.
-    - Deployment nodes can be used as input args in .bind() of other nodes in the DAG. 
+    - Deployment nodes can be used as input args in `.bind()` of other nodes in the DAG. 
     - Multiple nodes in the deployment graph naturally forms a DAG structure.
 - Accessing input:
     - Same input or output can be used in multiple nodes in the DAG.
     - Deployment nodes can access partial user input.
-- Dynamic control flow:
+- Dynamically controlling flow:
```
Comment (jiaodong) [inline doc/source/serve/deployment-graph.md:74]: nit: I think control flow is a programming language concept that is more accurate to stay as is

**Review by jiaodong — APPROVED** [review]
thanks for putting this up ! This indeed makes the grammar much better. Can you also ensure the linter passes ?

File: doc/source/serve/deployment-graph.md (hunk 1 above)
Comment (dmatrix) [reply doc/source/serve/deployment-graph.md:74]: No problem, make sense in that context. Will revert it back!

**Comment by dmatrix** [comment]
@jiaodong Not sure why the three tests are failing, but all others have passed, including the linter.

**Pull request closed without merging**